feature-toggles/user/user.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+function User(data) {
+    data = data || {};
+    _.extend(this, data);
+
+    if (!this.current_institution) {
+        this.changeInstitution();
+    }
+}
+
+User.prototype.changeInstitution = function changeInstitution(institution) {
```

Comment:
Will this be used?

Reply:
Yes, it is being used when creating the user, line 8.